nodejs/fastify/app/src/app.ts

```diff
@@ -1,10 +1,40 @@
 import Fastify from "fastify"
 import pino from "pino"
-import { Appsignal, AppsignalPinoTransport } from "@appsignal/nodejs"
+import { AppsignalPinoTransport } from "@appsignal/nodejs"
 
 const port = Number(process.env.PORT)
 
-const logger = pino(AppsignalPinoTransport({client: Appsignal.client, group: "application"}))
+// A Pino logger using the AppSignal Pino transport *as a destination*.
+function pinoAppsignalDestination() {
+  return pino(AppsignalPinoTransport({
+    group: "application"
```

Comment:
is this missing the client and that's why the CI fails?

```suggestion
    client: Appsignal.client,
    group: "application"
```

Reply:
No, it shouldn't need the client anymore, with the changes in https://github.com/appsignal/appsignal-nodejs/pull/1151. The CI fails because the changes in that PR have not been merged yet. With the changes in that PR, the test app does work, and CI should pass.